jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/list/AbstractMultiGridViewTest.java

```diff
@@ -275,28 +279,46 @@ public String getValue(T domain) {
     }
 
     @Test
-    public void testReloadColumnSortListWhenCatchColumnSortedEvent() {
-
+    public void testAddColumnSortHandler() {
         ListTable<T> listTable = mock(ListTable.class);
-        when(getView().getListGrid()).thenReturn(listTable);
-        GridPreferencesStore gridPreferencesStore = mock(GridPreferencesStore.class);
         when(listTable.getGridPreferencesStore()).thenReturn(gridPreferencesStore);
-        GridSortedColumnPreference gridSortedColumnPreference = new GridSortedColumnPreference("testOld", true);
-        when(gridPreferencesStore.getGridSortedColumnPreference()).thenReturn(gridSortedColumnPreference);
 
-        ColumnSortList columnSortList = getColumnSortList("startDate", null);
-        getColumnSortList("testNew", columnSortList);
+        doAnswer(invocation -> {
+            final ColumnSortEvent event = mock(ColumnSortEvent.class);
+            final ColumnSortList columnSortList = new ColumnSortList();
+            columnSortList.push(getColumn("test"));
+            when(event.getColumnSortList()).thenReturn(columnSortList);
+            ((ColumnSortEvent.Handler) invocation.getArguments()[0]).onColumnSort(event);
+            return null;
+        }).when(listTable).addColumnSortHandler(any());
 
```

Comment:
I think we need add " when(listTable.getColumnSortList()).thenReturn()" in here, avoid unit test failure.

Reply:
fixed